.github/workflows/swift-toolchain.yml

```diff
@@ -208,7 +203,7 @@ jobs:
 
   sqlite:
     needs: [context]
-    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    runs-on: "cirun-win11-x64-8--${{github.run_id}}"
```

Comment:
How does this work? The `--${{github.run_id}}` will expand to a unique runner name that is not listed in `.cirun.yml`

Reply:
This is a bit of magic to do with the cirun service. https://docs.cirun.io/reference/unique-runner-labels